api/src/opentrons/protocol_api/protocol_context.py

```diff
@@ -338,12 +338,20 @@ def load_labware(
         :param version: The version of the labware definition. You should normally
             leave this unspecified to let the implementation choose a good default.
         """
+        if isinstance(location, OffDeckType) and self._api_version < APIVersion(2, 15):
+            raise APIVersionError(
+                "Loading a labware off-deck is supported at apiLevel 2.15 and higher."
```

Comment:
To be a little clearer about what's going wrong:

```suggestion
                "Loading a labware off-deck requires apiLevel 2.15 or higher."
```